unit_testing/test_math.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
+# -*- coding: utf-8
+# pytest unit tests for spinalcordtoolbox.math
+
+
+from __future__ import absolute_import
+import sys
+import os
+import pytest
+import math
+import numpy as np
+from skimage.morphology import erosion, dilation, disk, ball, square, cube
+
+from spinalcordtoolbox.utils import __sct_dir__
+sys.path.append(os.path.join(__sct_dir__, 'scripts'))
+from spinalcordtoolbox import math
```

Comment:
> +./unit_testing/test_math.py:16: redefinition of unused 'math' from line 10

Reply:
oops! big mistake on my part. we now have python's `math` and `spinalcordtoolbox.math` packages which are both different. i changed my import for:
```python
import spinalcordtoolbox.math as sctmath
```
here: 2886a99ef9a3acbe82d963ad1a3113449b7c60bf

is that good practice?

Reply:
I think it doesn't matter so long as the naming is consistent across modules. This is one of the places Python's flexibility bites back. I saw elsewhere you used

```
import spinalcoretoolbox as sct

...
sct.math
```

I think that's the best option on the table.